tools/regs2groov.py

```diff
@@ -79,7 +79,14 @@ def generate_group(g):
 
         g = generate_group(defn)
         with open(f"{dest}/{name_func(defn.name)}.hpp", "w") as f:
-            print("#include <groov.hpp>", file=f)
+            print("#pragma once", file=f)
+
+            if includes is not None:
+                for include in includes:
+                    print(f"#include <{include}>", file=f)
```

Comment:
**suggestion**: Given a default empty list for includes, use a list comprehension + join here?
See https://github.com/intel/compile-time-init-build/blob/main/tools/gen_str_catalog.py#L144